qt_gui_cpp/src/qt_gui_cpp_sip/CMakeLists.txt

```diff
@@ -31,6 +31,39 @@ set(qt_gui_cpp_sip_INCLUDE_DIRS ${qt_gui_cpp_INCLUDE_DIRS} "${CMAKE_CURRENT_SOUR
 set(qt_gui_cpp_sip_LIBRARY_DIRS ${qt_gui_cpp_LIBRARY_DIRS} lib)
 set(qt_gui_cpp_sip_LDFLAGS_OTHER ${qt_gui_cpp_LDFLAGS_OTHER} -Wl,-rpath,\\"lib\\")
 
+find_library(class_loader_LIBRARY class_loader)
+find_library(console_bridge_LIBRARY console_bridge)
+find_library(TinyXML2_LIBRARY tinyxml2)
+
+get_filename_component(qt_gui_cpp_src_DIR ${CMAKE_CURRENT_BINARY_DIR} DIRECTORY)
+set(qt_gui_cpp_LIBRARY ${qt_gui_cpp_src_DIR}/qt_gui_cpp/Release/qt_gui_cpp${CMAKE_STATIC_LIBRARY_SUFFIX})
```

Comment:
Using a hard coded path like this (`Release`) won't work when e.g. built in `Debug`.

Reply:
You're right, it doesn't work. Working on a solution

Reply:
Has this been addressed? If yes, please comment here how / where so the reviewer doesn't have to search for it and mark the conversation as resolved.

Reply:
Yes this has now been addressed with generator expressions.